src/main/webapp/WEB-INF/tags/head.tag

```diff
@@ -1,2 +1,3 @@
 <%@tag description="Common head content" pageEncoding="UTF-8"%>
-<!-- This is where common <link> and <script> tags would go. -->
+<link href="style.css" rel="stylesheet">
+<script src="https://cse.google.com/cse.js?cx=017022763452466193978:ejddswiptcs"></script>
```

Comment:
With the `async` keyword, my previous comment said there might be a race condition if this script loaded before parsing the rest of the page. Without it, it makes the script synchronously downloaded, making the race condition always occur. But obviously that's not happening, so I took a closer look. It looks like your previous approach with `async` is from the [documentation](https://developers.google.com/custom-search/docs/tutorial/implementingsearchbox), and upon closer inspection, it looks like the script waits for the document to finish loading first, so `async` is fine after all.

Sorry for the back-and-forth.